src/components/mgt-person/mgt-person.ts

```diff
@@ -132,15 +149,24 @@ export class MgtPerson extends MgtTemplatedComponent {
   })
   public personCardInteraction: PersonCardInteraction;
 
-  @property({ attribute: false }) private isPersonCardVisible: boolean;
-  @property({ attribute: false }) private personCardShouldRender: boolean;
```

Comment:
This flag was used for lazy rendering the person card. With this change, the person card renders when the person renders which is not necessary. This become especially important when the person card will start making calls to the graph for additional data

Reply:
Good catch. I didn't realize that before I trimmed it. I'll add it back